schemas/concrete/MPTitle.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$id": "MPTitle.json",
+    "title": "Title",
+    "$mash": {
+      "sources": [
+        { "$ref": "abstract/MPModel.json#" },
+        { "$ref": "mixins/MPContainedDocument.json#" },
+        { "$ref": "mixins/MPManuscriptDocument.json#" }
+      ],
+      "with": {
+        "properties": {
+          "articleTitle": {
```

Comment:
If we change the object type to "MPTitles", let's also change this to "title".

Reply:
done